docs/tools_user_guide.md

```diff
@@ -262,9 +266,7 @@ Also available as API arguments:
 - `benchmark_script(backend=...)`
 - `benchmark_model(backend=...)`
 
-### Runtime (future feature)
-
-*Future feature, not yet implemented.* 
+### Runtime
```

Comment:
```suggestion
### Runtimes
```

and anywhere else in the docs where the flag/arg is named `runtime` instead of `runtimes`. Note: `--runtime` is a legal arg for use in examples, thanks to argparse magic.

Reply:
Done everywhere, except in `benchmark_model`, since only one runtime can be received at a time there.